scriptmodules/supplementary/splashscreen.sh

```diff
@@ -27,20 +27,41 @@ function _video_exts_splashscreen() {
 }
 
 function depends_splashscreen() {
-    getDepends fbi omxplayer
+    getDepends fbi omxplayer insserv
 }
 
 function install_bin_splashscreen() {
-    cp "$md_data/asplashscreen" "/etc/init.d/"
+    cp -a "$md_data/asplashscreen.sh" "$rootdir/lib/"
```

Comment:
`"$md_inst"` would be a more suitable location (actual install folder for the module)